Documentation/gettingstarted/aws.rst

```diff
@@ -0,0 +1,303 @@
+        image: mesosphere/aws-cli
+        command: ['sh', '-c', 'sleep 3600']
+        env:
+          - name: AWS_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                name: cilium-aws
+                key: AWS_ACCESS_KEY_ID
+                optional: true
+          - name: AWS_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: cilium-aws
+                key: AWS_SECRET_ACCESS_KEY
+                optional: true
+          - name: AWS_DEFAULT_REGION
+            valueFrom:
+              secretKeyRef:
+                name: cilium-aws
+                key: AWS_DEFAULT_REGION
+                optional: true
+
+To list all the AWS instances available the following command can be used:
+
+.. parsed-literal::
+
+   $ kubectl  -n kube-system exec -ti testing-aws-pod -- aws ec2 describe-instances
+
+.. include:: cilium_install.rst
+
+It is important for this demo that ``kube-dns`` is working correctly. To know the
+status of ``kube-dns`` you can run the following command:
+::
+
+    $ kubectl get deployment kube-dns -n kube-system
+    NAME       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    kube-dns   1         1         1            1           13h
+
+Where at least one pod should be available.
+
+
+Step 2: Create a sample policy
+================================
+
+Deploy a demo application:
+----------------------------
+
+In this case we're going to use a demo application that it is used in other
+guides. Example manifest will create three microservices applications:
+*deathstar*, *tiefighter*, and *xwing*. In this case, we are only going to use
+our *xwing* microservice to secure communications to existing AWS instances.
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/http-sw-app.yaml
+    service "deathstar" created
+    deployment "deathstar" created
+    deployment "tiefighter" created
+    deployment "xwing" created
+
+
+Kubernetes will deploy the pods and service in the background.  Running
+``kubectl get pods,svc`` will inform you about the progress of the operation.
+Each pod will go through several states until it reaches ``Running`` at which
+point the pod is ready.
+
+::
+
+    $ kubectl get pods,svc
+    NAME                             READY     STATUS    RESTARTS   AGE
+    po/deathstar-76995f4687-2mxb2    1/1       Running   0          1m
+    po/deathstar-76995f4687-xbgnl    1/1       Running   0          1m
+    po/tiefighter                    1/1       Running   0          1m
+    po/xwing                         1/1       Running   0          1m
+
+    NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+    svc/deathstar    ClusterIP   10.109.254.198   <none>        80/TCP    3h
+    svc/kubernetes   ClusterIP   10.96.0.1        <none>        443/TCP   3h
+
+Policy Language:
+-----------------
+
+To enforce security to AWS instances, the **ToGroups** rule will be used. This
```

Comment:
I'd rephrase this since we currently only interface with AWS. What about: 

"**ToGroups** rules can be used to define policy in relation to cloud providers, like AWS."